modules/ximgproc/src/deriche_filter.cpp

```diff
@@ -0,0 +1,410 @@
+#include "precomp.hpp"
```

Comment:
Please add license header into this source file (license header is not required for samples, and small utility files like `precomp.hpp`).